### PR TITLE
Improved chart styling and events, other window improvements

### PR DESCRIPTION
**Pull request by mirakst**
Main window no longer has ghosting on resize.

Improved maximize function.

Charts have better min/max values now.

Chart zoom modes fixed.

Improved latest cpu & ram reading visuals.

Fixed ghosting in manager module. (main window and detached)

### CODE REVIEW

File: DashboardFrontend/Charts/NetworkChart.cs [hunk 1]
```diff
@@ -72,10 +72,10 @@ public NetworkChart()
                 {
                     Name = "SendReceived",
                     Labeler = (value) => value.ToString("N0") + "GB",
-                    MinLimit = 0,
-                    MaxLimit = 100,
                     LabelsPaint = new SolidColorPaint(new SKColor(255, 255, 255)),
                     SeparatorsPaint = new SolidColorPaint(new SKColor(255, 255, 255)), 
+                    MinLimit = 0,
+                    MaxLimit = 100,
```
Comment (AndersMariegaard) [inline DashboardFrontend/Charts/NetworkChart.cs:78]: Virker ikke til at gøre noget det her

File: DashboardFrontend/DetachedWindows/HealthReportDetached.xaml.cs [hunk 2]
```diff
@@ -13,26 +17,28 @@ public HealthReportDetached(HealthReportViewModel healthReportViewModel)
...
+            Vm.NetworkDeltaChart.ChangeMaxView(comboBoxItemValue);
+            Vm.NetworkSpeedChart.ChangeMaxView(comboBoxItemValue);
         }
 
         private void CartesianChart_MouseLeave(object sender, System.Windows.Input.MouseEventArgs e)
         {
-            Vm.SystemLoadChart.AutoFocusOn();
-            Vm.NetworkChart.AutoFocusOn();
+            DataChart? ass = (DataChart)(sender as CartesianChart)?.DataContext!;
+            ass?.AutoFocusOn();
```
Comment (AndersMariegaard) [inline DashboardFrontend/DetachedWindows/HealthReportDetached.xaml.cs]: sus

File: DashboardFrontend/DetachedWindows/HealthReportDetached.xaml.cs [hunk 3]
```diff
@@ -13,26 +17,28 @@ public HealthReportDetached(HealthReportViewModel healthReportViewModel)
...
         }
 
         private void CartesianChart_MouseEnter(object sender, System.Windows.Input.MouseEventArgs e)
         {
-            Vm.SystemLoadChart.AutoFocusOff();
-            Vm.NetworkChart.AutoFocusOff();
+            DataChart? ass = (DataChart)(sender as CartesianChart)?.DataContext!;
+            ass?.AutoFocusOff();
```
Comment (AndersMariegaard) [inline DashboardFrontend/DetachedWindows/HealthReportDetached.xaml.cs]: sus x2

File: DashboardFrontend/MainWindow.xaml.cs [hunk 4]
```diff
@@ -149,22 +156,24 @@ private void DraggableGrid(object sender, MouseButtonEventArgs e)
 
         private void CartesianChart_MouseLeave(object sender, MouseEventArgs e)
         {
-            ViewModel.HealthReportViewModel.SystemLoadChart.AutoFocusOn();
+            DataChart? ass = (DataChart)(sender as CartesianChart)?.DataContext!;
+            ass?.AutoFocusOn();
```
Comment (AndersMariegaard) [inline DashboardFrontend/MainWindow.xaml.cs]: åh oh

File: DashboardFrontend/MainWindow.xaml.cs [hunk 5]
```diff
@@ -149,22 +156,24 @@ private void DraggableGrid(object sender, MouseButtonEventArgs e)
...
-            ViewModel.HealthReportViewModel.SystemLoadChart.AutoFocusOn();
+            DataChart? ass = (DataChart)(sender as CartesianChart)?.DataContext!;
+            ass?.AutoFocusOn();
         }
 
         private void CartesianChart_MouseEnter(object sender, MouseEventArgs e)
         {
-            ViewModel.HealthReportViewModel.SystemLoadChart.AutoFocusOff();
+            DataChart? ass = (DataChart)(sender as CartesianChart)?.DataContext!;
+            ass?.AutoFocusOff();
```
Comment (AndersMariegaard) [inline DashboardFrontend/MainWindow.xaml.cs]: åh oh x4

**Review by AndersMariegaard — CHANGES_REQUESTED** [review]
Dismantle ass

**Review by TorbjoernH — APPROVED** [review]

**Pull request merged**